src/main/app/cypress/integration/hakuForm/createHakuForm.ts

```diff
@@ -87,14 +87,25 @@ const fillAikatauluSection = () => {
       });
     });
 
-    getByTestId('alkamiskausi').within(() => {
+    cy.findByText('hakulomake.alkamiskausi').click();
+
+    getByTestId('KausiJaVuosiFields').within(() => {
       getRadio('kausi_0#1').click({ force: true });
+      selectOption(new Date().getFullYear().toString());
+
+      cy.findByText('hakulomake.tiedossaTarkkaAjankohta').click();
 
-      getByTestId('vuosi').click();
+      getByTestId('alkaa').within(() => {
```

Comment:
Jäin miettimään miks redux-formin kanssa tarvii käyttää testIdtä kentän selectorissa mutta ilmeisesti noille generoituu jotain random id-arvoja? Pitää tutustua miten tää kouta-UI pyörähtää

Reply:
Varmasti on parempikin tapa tehdä tämä, mutta käsittääkseni noilla ei ole muita id-arvoja, joita voisi tässä käyttää. Labelin käyttäminen vaatisi jonkin verran refaktorointia.